docs/user-guide/petset.md

```diff
@@ -131,6 +131,8 @@ pvc-902733c2-3717-11e6-a46e-42010af00002   1Gi        RWO           Bound     de
 
 The network identity has 2 parts. First, we created a headless Service that controls the domain within which we create Pets. The domain managed by this Service takes the form: `$(service name).$(namespace).svc.cluster.local`, where "cluster.local" is the [cluster domain](http://releases.k8s.io/{{page.githubbranch}}/build/kube-dns/README.md#how-do-i-configure-it). As each pet is created, it gets a matching DNS subdomain, taking the form: `$(petname).$(governing service domain)`, where the governing service is defined by the `serviceName` field on the Pet Set.
 
+Here are some examples of choices for Cluster Domain, Service name, Pet Set name, and how that affects the DNS names for the Pets and the hostnames in the Pet's pods. Note that Cluster Domain defaults to `cluster.local`, but you might've changed it by doing [this](http://releases.k8s.io/{{page.githubbranch}}/build/kube-dns/README.md#how-do-i-configure-it):
```

Comment:
I would move that note to be underneath the table and change the wording to, simply:

Note: The Cluster Domain will `cluster.local` unless [otherwise configured](http://releases.k8s.io/{{page.githubbranch}}/build/kube-dns/README.md#how-do-i-configure-it).